api/src/wfl/api/routes.clj

```diff
@@ -43,11 +42,11 @@
             :swagger {:tags ["Informational"]}}}]
    ["/oauth2id"
     {:get {:summary   "Get the OAuth2 Client ID for this deployment of the server"
-           :handler   (fn [_] (handlers/succeed {:oauth2-client-id @once/oauth-client-id}))
+           :handler   (fn [_] (handlers/succeed {:oauth2-client-id (env/getenv "WFL_OAUTH2_CLIENT_ID")}))
            :responses {200 {:body {:oauth2-client-id string?}}}
            :swagger   {:tags ["Informational"]}}}]
-   ["/api/v1/environments"
-    {:get  {:summary "Get all of the environments the server knows"
+   ["/api/v1/environment"
+    {:get  {:summary "Get all of the environment the server knows"
             :parameters nil
             :responses {200 {:body map?}}
             :handler (handlers/success env/stuff)}}]
```

Comment:
Will this expose `GOOGLE_APPLICATION_CREDENTIALS` and `WFL_COOKIE_SECRET`?
Maybe: `:summary "Show relevant variables deployed in the process environment."`

Reply:
Actually this endpoint is not that useful anymore since it shows only the `debug` map which is simply just:

```
{:debug {:data-repo {:service-account "jade-k8-sa@broad-jade-dev.iam.gserviceaccount.com",
                     :url "https://jade.datarepo-dev.broadinstitute.org"},
         :google {:jes_roots ["gs://broad-gotc-dev-cromwell-execution"], :projects ["broad-gotc-dev"]},
         :server {:project "broad-gotc-dev",
                  :service-account "secret/dsde/gotc/dev/wfl/wfl-non-prod-service-account.json",
                  :vault "secret/dsde/gotc/dev/zero"},
         :doc "A local environment for development and debugging.",
         :keyword :debug,
         :name "debug"}}
```
I guess we could just take this endpoint out?

Reply:
OK by me.

Reply:
done. Later on if we need to look at the default options and inputs of a module, we could add an endpoint to expose them.